docs/topics/db/fixtures.txt

```diff
@@ -40,8 +40,8 @@ or to provide some :ref:`initial data <initial-data-via-fixtures>` using the
 
     django-admin loaddata <fixture label>
 
-Where Django looks for fixtures?
-================================
+Where does Django look for fixtures?
```

Comment:
I would propose we remove all question marks, so for this remaining case, how about:
```suggestion
How fixtures are discovered
```